client-src/elements/chromedash-featurelist.ts

```diff
@@ -8,48 +10,47 @@ const MAX_FEATURES_SHOWN = 500;
 class ChromedashFeaturelist extends LitElement {
   static styles = FEATURELIST_CSS;
 
-  static get properties() {
-    return {
-      user: {type: Object},
-      isSiteEditor: {type: Boolean},
-      signedInUser: {type: String},
-      editableFeatures: {type: Object},
-      features: {attribute: false}, // Directly edited and accessed in template/features.html
-      metadataEl: {attribute: false}, // The metadata component element. Directly edited in template/features.html
-      searchEl: {attribute: false}, // The search input element. Directly edited in template/features.html
-      filtered: {attribute: false},
-      openFeatures: {attribute: false},
-      starredFeatures: {attribute: false},
-    };
-  }
-
-  constructor() {
-    super();
-    this.user = {};
-    this.features = [];
-    this.editableFeatures = [];
-    this.filtered = [];
-    this.metadataEl = document.querySelector('chromedash-metadata');
+  @property({type: Object})
+  user!: User;
+  @property({type: Boolean})
+  isSiteEditor = false;
+  @property({type: String})
+  signedInUser = '';
+  @property({type: Array})
+  editableFeatures!: Feature[];
```

Comment:
I believe this type should be 
```suggestion
  editableFeatures!: number[];
```
Because it's set at https://github.com/GoogleChrome/chromium-dashboard/blob/55e6a2b8c07d92995541d970e0d9a983d971acca/templates/features.html#L35, which can't assign an object type. (On the other hand, that line appears to come from https://github.com/GoogleChrome/chromium-dashboard/blob/55e6a2b8c07d92995541d970e0d9a983d971acca/framework/basehandlers.py#L497 and so always be an empty array. But this PR isn't the place to fix that. FYI @DanielRyanSmith)